cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1286,6 +1286,12 @@ function Start-WorkerServices {
   Log-Output "Kubernetes components started successfully"
 }
 
+# Stop and unregister both kubelet & kube-proxy services.
+function Unregister-WorkerServices {
+  & sc.exe delete kubelet
+  & sc.exe delete kube-proxy
```

Comment:
It probably doesn't make any difference, but we [start](https://github.com/kubernetes/kubernetes/blob/cd426eaee6d4ceaaa434b1eca8cbf5c168305522/cluster/gce/windows/k8s-node-setup.psm1#L1261) the kubelet before the kube-proxy, so it would make more sense to reverse the order: stop + unregister kube-proxy, then kubelet.

Reply:
Reversed.